.github/workflows/ci-cd.yml

```diff
@@ -506,37 +506,36 @@ jobs:
         - macos-latest
         - windows-latest
         python-version:
-        - 3.11
+        - "3.11"
         - pypy-3.9
-        - 3.6
-        - 3.8
-        - 3.7
-        - 3.9
-        - >-
-          3.10
+        - "3.6"
+        - "3.8"
+        - "3.7"
+        - "3.9"
+        - "3.10"
         - pypy-3.6
-        - ~3.12.0-0
```

Comment:
Could probably add the same for 3.13.

Reply:
I think that's better for a separate PR.